src/adapter/adapter.js

```diff
@@ -1,7 +1,14 @@
 // @flow
 import youbora from 'youboralib';
 import {core} from 'kaltura-player-js';
-const {Error, MediaType} = core;
+const {Error, MediaType, DrmScheme} = core;
+
+const DrmSchemeTitle = {
+  FAIRPLAY: 'FairPlay',
+  PLAYREADY: 'PlayReady',
+  WIDEVINE: 'Widevine',
```

Comment:
```suggestion
  PLAYREADY: 'PlayReadyCENC',
  WIDEVINE: 'WidevineCENC',
```
per https://github.com/kaltura/playkit-android/blob/e8b421cce3222d7eb18ef5fb01e24cbc64f2a642/playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java#L25